activities/communication/using-chat.md

```diff
@@ -0,0 +1,66 @@
+---
+type: guide
+---
+
+# How to use [chat.publiccode.net](https://chat.publiccode.net/)
+
+We use Mattermost, hosted at chat.publiccode.net for informal chat purposes.
+
+Mattermost lets us create multiple teams with different members. We only have 1 team right now: our staff team.
+
+## Staff team
+
+Discussions on chat.publiccode.net are:
+
+* timely
+* optional
+* secret
+
+If something must be known by all staff, please send an email.
```

Comment:
Sorry for being unclear. My point was rather that I believe email is what we should use even if it is just one person that need to no something, and that chat is not to be used for such important messages even if the message are not meant for multiple staff members.

Reply:
Got it. I've fixed that now.